addons/website/static/src/services/website_custom_menus.js

```diff
@@ -137,4 +137,4 @@ registry.category('website_custom_menus').add('website.custom_menu_edit_menu', {
     isDisplayed: (env) => env.services.website.currentWebsite
         && env.services.website.currentWebsite.metadata.contentMenus
         && env.services.website.currentWebsite.metadata.contentMenus.length,
-});
+}, { force: true });
```

Comment:
Can you please explain the need for this change?

Reply:
IMO, the `custom_menu_edit_menu` is used in other modules and we're updating in website_event, so I think when other modules trying to use that it causes `registry: it already exists` error, so I have added `force: true` here.
Ref - https://runbot.odoo.com/runbot/build/68368173